wear/src/main/java/chat/rocket/android/util/String.kt

```diff
@@ -0,0 +1,21 @@
+package chat.rocket.android.util
+
+fun String.removeTrailingSlash(): String {
```

Comment:
This is duplicate from the existing extension function defined in the `app` module. Please, remove.